templates/includes/related-articles-highlight-cards.html

```diff
@@ -26,24 +26,25 @@ <h3 class="highlight-cards__card_title">{{ article.title }}</h3>
                             <p class="highlight-cards__description">{{ article.teaser_text }}</p>
                         </div>
                     </a>
+                {% else %}
```

Comment:
This was a logical error I found when looking through the QS results to fix the above naming convention error, we used the first article twice in the cards. I didn't notice as I was using generic test data.